pwcnet/feature_pyramid_network/model_test.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+import tensorflow as tf
+import unittest
+from pwcnet.feature_pyramid_network.model import FeaturePyramidNetwork
+from tensorflow.contrib.layers import l2_regularizer
+
+
+class TestFeaturePyramid(unittest.TestCase):
+    def setUp(self):
+        self.feature_pyr_net = FeaturePyramidNetwork(regularizer=l2_regularizer(1e-4))
+
+        config = tf.ConfigProto()
+        config.gpu_options.allow_growth = True
+        self.sess = tf.Session(config=config)
+
+    def test_network(self):
+        """
+        Sets up the network's forward pass and ensures that all shapes are expected.
+        """
+        height = 512
+        width = 512
```

Comment:
To help my weak laptop run all the tests faster, can we make this something like 64 or 128 instead (as small as possible)?